style.css

```diff
@@ -0,0 +1,296 @@
+body {
+  margin: 0;
+  background: #181818;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 400;
+  color: #b4b6ba;
+  font-style: normal;
+}
+
+h2 {
+  color: white;
+  width: 100%;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.logo-box {
+  display: flex;
+  justify-content: center;
+}
+
+.logo {
+  background: url(https://cdn.freebiesupply.com/logos/large/2x/movie-city-1-logo-black-and-white.png);
+  background-size: 2em;
+  background-repeat: no-repeat;
+  text-indent: 10000%;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-bottom: 1em;
+}
+
+.logo:hover {
+  cursor: pointer;
+}
+
+.content {
+  /*background-image: url(https://gallery.yopriceville.com/var/resizes/Backgrounds/Cinema_Background.jpg?m=1399672800);*/
+  padding-top: 6em;
+  background-blend-mode: multiply;
+  display: grid;
+  place-items: center;
+  background-size: contain;
+}
+
+header {
+  background: #202020;
+  text-align: center;
+  position: fixed;
+  width: 100%;
+}
+
+/*Nav Bar*/
+
+.nav-toggle {
+  display: none;
+}
+
+.nav-toggle-label {
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin-left: 1em;
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.nav-toggle-label span,
+.nav-toggle-label span::before,
+.nav-toggle-label span::after {
+  background: white;
+  display: block;
+  height: 2px;
+  width: 2em;
+  border-radius: 2px;
+  position: relative;
+}
+
+.nav-toggle-label span::before,
+.nav-toggle-label span::after {
+  content: "";
+  position: absolute;
+}
+
+.nav-toggle-label span::before {
+  bottom: 7px;
+}
+
+.nav-toggle-label span::after {
+  top: 7px;
+}
+
+nav {
+  position: absolute;
+  text-align: left;
+  top: 100%;
+  left: 0;
+  background: #242424;
+  width: 100%;
+  transform-origin: top;
+  transform: scale(1, 0);
+  transition: transform 400ms ease-in-out;
```

Comment:
group similar property so other developer can read your document easily.

Reply:
please be more specific, I'm unable to understand the issue.

Reply:
great informative link, will apply those changes. thanks mansoor bhai.